packages/sanity/playwright-ct/tests/formBuilder/inputs/PortableText/Input.spec.tsx

```diff
@@ -37,7 +37,7 @@ test.describe('Portable Text Input', () => {
       // Write some text
       await insertPortableText('Hello there', $pte)
       // Assertion: placeholder was removed
-      expect(await $placeholder.count()).toEqual(0)
+      expect($placeholder).toHaveCount(0)
```

Comment:
Looks good, just have to wait for this async assertion

```suggestion
      await expect($placeholder).toHaveCount(0)
```

Reply:
Nice catch!